orchestrator/rpc/api/events/event_api.go

```diff
@@ -8,6 +8,110 @@ import (
 	"github.com/pkg/errors"
 )
 
+// MinimalConsensusInfo
+func (api *PublicFilterAPI) MinimalConsensusInfo(ctx context.Context, requestedEpoch uint64) (*rpc.Subscription, error) {
+	notifier, supported := rpc.NotifierFromContext(ctx)
+	if !supported {
+		return &rpc.Subscription{}, rpc.ErrNotificationsUnsupported
+	}
+	rpcSub := notifier.CreateSubscription()
+
+	go func() {
+
+		batchSender := func(start, end uint64) error {
+			epochInfos, err := api.backend.ConsensusInfoByEpochRange(start)
+			if err != nil {
+				log.WithError(err).Error("Some epoch infos are missing in db.")
+				return errors.Wrap(err, "Missing epoch infos in db. Could not send over stream.")
+			}
+			latestFinalizedSlot := api.backend.LatestFinalizedSlot()
+			for _, ei := range epochInfos {
+				if err := notifier.Notify(rpcSub.ID, &generalTypes.MinimalEpochConsensusInfoV2{
+					Epoch:            ei.Epoch,
+					ValidatorList:    ei.ValidatorList,
+					EpochStartTime:   ei.EpochStartTime,
+					SlotTimeDuration: ei.SlotTimeDuration,
+					FinalizedSlot:    latestFinalizedSlot,
+				}); err != nil {
+					log.WithField("start", start).
+						WithField("end", end).
+						WithError(err).
+						Error("Failed to send epoch info. Could not send over stream.")
+					return errors.Wrap(err, "Failed to send epoch info. Could not send over stream.")
+				}
+				log.WithField("epoch", ei.Epoch).WithField("latestFinalizedSlot", latestFinalizedSlot).
+					Info("published epoch info to pandora")
```

Comment:
Inconsistent formatting

```suggestion
					Info("Published epoch info to pandora")
```